src/Persistence/Mapping/Driver/FileDriver.php

```diff
@@ -71,7 +71,7 @@ public function getGlobalBasename(): string
      *
      * @throws MappingException
      */
-    public function getElement(string $className): ClassMetadata
+    public function getElement(string $className)
```

Comment:
When running the ORM 3 test suite, line 95 causes lots of issues:

`Doctrine\Persistence\Mapping\Driver\FileDriver::getElement(): Return value must be of type Doctrine\Persistence\Mapping\ClassMetadata, SimpleXMLElement returned`

Reply:
to me, this looks like invalid tests